src/Our.Umbraco.Ditto/ComponentModel/TypeConverters/HtmlStringConverter.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace Our.Umbraco.Ditto
+{
+    using System;
+    using System.ComponentModel;
+    using System.Globalization;
+    using System.Web;
+
+    using global::Umbraco.Web;
+    using global::Umbraco.Web.Templates;
+
+    /// <summary>
+    /// Provides a unified way of converting <see cref="String"/>s to <see cref="HtmlString"/>'s.
+    /// </summary>
+    public class HtmlStringConverter : TypeConverter
+    {
+        /// <summary>
+        /// Returns whether this converter can convert an object of the given type to the type of this converter,
+        /// using the specified context.
+        /// </summary>
+        /// <param name="context">
+        /// An <see cref="T:System.ComponentModel.ITypeDescriptorContext" /> that provides a format context.
+        /// </param>
+        /// <param name="sourceType">
+        /// A <see cref="T:System.Type" /> that represents the type you want to convert from.
+        /// </param>
+        /// <returns>
+        /// true if this converter can perform the conversion; otherwise, false.
+        /// </returns>
+        public override bool CanConvertFrom(ITypeDescriptorContext context, Type sourceType)
+        {
+            if (sourceType == typeof(string) || sourceType == typeof(HtmlString))
+            {
+                return true;
+            }
+
+            return base.CanConvertFrom(context, sourceType);
+        }
+
+        /// <summary>
+        /// Converts the given object to the type of this converter, using the specified context and culture information.
+        /// </summary>
+        /// <param name="context">An <see cref="T:System.ComponentModel.ITypeDescriptorContext" /> that provides a format context.</param>
+        /// <param name="culture">The <see cref="T:System.Globalization.CultureInfo" /> to use as the current culture.</param>
+        /// <param name="value">The <see cref="T:System.Object" /> to convert.</param>
+        /// <returns>
+        /// An <see cref="T:System.Object" /> that represents the converted value.
+        /// </returns>
+        public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, object value)
+        {
+            if (value == null)
+            {
+                return null;
+            }
+
+            string s = value.ToString();
+            if (value is HtmlString)
```

Comment:
There must be something converting it deeper in the core as it's coming through as one when I test it.